### PR TITLE
New Rule: Link Metamask Impersonation with suspicious link

### PR DESCRIPTION
**Pull request by morriscode**

### CODE REVIEW

File: link_metamask_impersonation_phishing_suspicious_link.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+name: "Phishing: MetaMask Impersonation with Suspicious Links"
+description: |
+  Detects phishing attempts impersonating metamask, with suspicious link(s)
+references:
+  - "https://metamask.zendesk.com/hc/en-us/articles/12683145255835-I-received-an-email-claiming-to-be-from-MetaMask-Is-it-legit-"
+type: "rule"
+severity: "high"
```
Comment (morriscode) [inline link_metamask_impersonation_phishing_suspicious_link.yml:7]:
```suggestion
severity: "medium"
```

File: link_metamask_impersonation_phishing_suspicious_link.yml [hunk 2]
```diff
@@ -0,0 +1,25 @@
+name: "Phishing: MetaMask Impersonation with Suspicious Links"
+description: |
+  Detects phishing attempts impersonating metamask, with suspicious link(s)
+references:
+  - "https://metamask.zendesk.com/hc/en-us/articles/12683145255835-I-received-an-email-claiming-to-be-from-MetaMask-Is-it-legit-"
+type: "rule"
+severity: "medium"
+source: |
+  type.inbound
+  and any([body.html.display_text, sender.display_name, subject.subject],
```
Comment (morriscode) [inline link_metamask_impersonation_phishing_suspicious_link.yml:10]: ### 
```suggestion
  and any([sender.display_name, subject.subject],
```

Comment (morriscode) [reply link_metamask_impersonation_phishing_suspicious_link.yml:10]: Found an FP

File: link_metamask_impersonation_phishing_suspicious_link.yml [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+type: "rule"
+severity: "medium"
+source: |
+  type.inbound
+  and any([sender.display_name, subject.subject],
+    regex.icontains(., '[MМⅯ𝕄𝔐𝗠𝙈𝙢𝙼𝚖][eⅴ𝑒𝒆𝓔𝔢𝕖𝕖𝖊𝖊𝗲𝘀𝙀𝚎𝜅𝝴𝞮][t𝑇𝒕𝓣𝔗𝕥𝕥𝖙𝖙𝘵𝚃𝛕𝝉𝞃𝞣][aaₐ𝖆𝗮𝘢𝙖𝚊𝛂𝜶𝞪]\s?[MМⅯ𝕄𝔐𝗠𝙈𝙢𝙼𝚖][aaₐ𝖆𝗮𝘢𝙖𝚊𝛂𝜶𝞪][ѕs𝑆𝒔𝓢𝔰𝕊𝕊𝖘𝖘𝗦𝘴𝙎𝚂][k𝑘𝒌𝔞𝕜𝕜𝗞𝘬𝙆𝙉𝙠]')
+  )
+  and not sender.email.domain.domain in ("metamask.io", "zendesk.com", "discoursemail.com")
+  and any(body.links,
+    any([beta.linkanalysis(.)],
```
Comment (jkamdjou) [inline link_metamask_impersonation_phishing_suspicious_link.yml:15]: sadly `linkanalysis()` rules currently don't run during NUX right now 😞 do you think there's a version of this rule that we could use without that function? all good if not

Comment (morriscode) [reply link_metamask_impersonation_phishing_suspicious_link.yml:15]: Let me see what we can do!

**Comment by morriscode** [comment]
This has been running for 2 weeks. It has 4 hits, and should be fairly safe to deploy.

**Review by aidenmitchell — APPROVED** [review]

**Comment by morriscode** [comment]
Closing may want to approach this via logodetect

**Pull request closed without merging**